python/lsst/summit/extras/assessQFM.py

```diff
@@ -0,0 +1,238 @@
+        """Compare a table of quickFrameMeasurement results with the
+        baseline vetted data, and print output of the comparison.
+
+        Parameters
+        ----------
+        comparisonData : `pandas.DataFrame`
+            Table to compare with baseline results.
+        """
+        baselineData = self.testData.loc[comparisonData.index]
+
+        # First the cases that succeeded in the baseline results:
+        for key in ['G', 'QG', 'WF', 'DG', 'SG', 'FP', 'U']:
+            key_inds = baselineData['finalTag'] == key
+            if key_inds.sum() == 0:
+                continue
+            origResults = baselineData[key_inds]
+            newResults = comparisonData[key_inds]
+
+            stillSucceeds = (newResults['finalTag'] == 'P').sum()
+            print(f"Results for '{self.resultKey[key]}' cases:")
+            print(f"    {stillSucceeds} out of {len(origResults)} still succeed")
+
+            centroid_distances = ((origResults['centroid_x'] - newResults['centroid_x'])**2 +
+                                  (origResults['centroid_y'] - newResults['centroid_y'])**2)**0.5
+
+            if key in ['G', 'QG', 'DG']:
+                inCut = centroid_distances < self.cuts[key]
+                print(f"    {inCut.sum()} out of {len(origResults)} centroids are within {self.cuts[key]} "
+                      "pixels of the baseline centroid fit.")
+            if key in ['U', 'WF', 'QG']:
+                print("    Individual exposures:")
+                print(f"    {'day_obs':<10}{'sequence_number':<17}{'old centroid':<17}{'new centroid':<17}")
+                for i, res in origResults.iterrows():
+                    newRes = newResults.loc[i]
+                    old_centroid = f"({res['centroid_x']:.1f}, {res['centroid_y']:.1f})"
+                    new_centroid = f"({newRes['centroid_x']:.1f}, {newRes['centroid_y']:.1f})"
+                    print(f"    {res['day_obs']:<10}{res['sequence_number']:<17}{old_centroid:<17}"
+                          f"{new_centroid:<17}")
+
+        # Next the cases that failed in the past:
+        for key in ['FG', 'DF', 'SF', 'OF']:
+            key_inds = baselineData['finalTag'] == key
+            if key_inds.sum() == 0:
+                continue
+            origResults = baselineData[key_inds]
+            newResults = comparisonData[key_inds]
+
+            stillFails = (newResults['finalTag'] == 'F').sum()
+            print(f"Results for '{self.resultKey[key]}' cases:")
+            print(f"    {stillFails} out of {len(origResults)} still fail")
+
+        print("Runtime comparison:")
+        print(f"    Baseline: {np.mean(baselineData['runtime']):.2f}+/-"
+              f"{np.std(baselineData['runtime']):.2f} seconds")
+        print(f"    Current: {np.mean(comparisonData['runtime']):.2f}+/-"
+              f"{np.std(comparisonData['runtime']):.2f} seconds")
+
+
+if __name__ == '__main__':
+
+    butler = butlerUtils.makeDefaultLatissButler(embargo=True)
```

Comment:
Could you put some of these options into an `argparse` instead of hard-coding them?

Reply:
Yes, good suggestion.